Tools/CISupport/ews-build/steps.py

```diff
@@ -355,6 +355,10 @@ def getResultSummary(self):
         else:
             return {'step': 'Cleaned and updated working directory'}
 
+    def run(self):
```

Comment:
Can you please explain what was old and new behavior here?

Reply:
https://ews-build.webkit-uat.org/#/builders/9/builds/3273 is a good example of this working on a branch PR, you can see that our `got_revision` is actually on a branch